rest/admin-ui-ext/src/main/java/org/keycloak/admin/ui/rest/SessionsResource.java

```diff
@@ -75,8 +73,7 @@ public Stream<SessionRepresentation> realmSessions(@QueryParam("type") @DefaultV
                     .keySet().stream().map(i -> new SessionId(i, OFFLINE)));
         }
 
-
-        final List<SessionId> sessionIds = clientIds.skip(first).limit(max).collect(Collectors.toList());
+        final List<SessionId> sessionIds = clientIds.collect(Collectors.toList());
```

Comment:
Is this transformation into a List<> really necessary here? As far as I can see, it is just iterated over so it could just be used as a stream...

Reply:
you are right, changed that